src/core/services.py

```
@@ -81,61 +76,143 @@ def run(
-            group_capacities=group_capacities,
-            score_weights=score_weights,
-            opt_mode=opt_mode,
-            conflict_priority=conflict_priority,
-            timeout_seconds=timeout_seconds,
-        )
 
-        try:
             return solver_interface.run_optimization(
                 participants, cfg, status_box=status_box
             )
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)